scipy/optimize/__init__.py

```diff
@@ -422,7 +422,7 @@
                            LinearConstraint,
                            Bounds)
 from ._hessian_update_strategy import HessianUpdateStrategy, BFGS, SR1
-from ._shgo import shgo
+from ._shgo import shgo, SHGO
```

Comment:
```suggestion
from ._shgo import shgo
```
This is causing the final refguide failure, I think. What was the intent here? I'll remove it if I find locally that it's not being imported anywhere.

Reply:
This is leftover from another branch update where `SHGO.HC.save_complex` and `SHGO.HC.load_complex` is used to save and load the progress of the optimization run to and from the hard disk drive, which is needed on clusters where user runtime is limited.

Currently there aren't many users that use this feature yet (and the code is not well tested), so I will make a separate PR in the future, it should also be possible to make a cleaner implementation for the `shgo` function to avoid a second class object in the namespace.